### PR TITLE
Integrate Dashboard/Request for Access with the Backend [Continued]

### PR DESCRIPTION
**Pull request by KentShikama**
Continued from https://github.com/codeforpdx/dwellingly-app/pull/172

Related to https://github.com/codeforpdx/dwellingly-app/issues/163

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -2,6 +2,7 @@
   "name": "dwellingly",
   "version": "0.1.0",
   "private": true,
+  "proxy": "http://localhost:5000",
```
Comment (Chris-Boe) [inline package.json:5]: We already have our URL served in an environment variable, which Heroku will be transforming to a different URL for other environments

**Review by Chris-Boe — CHANGES_REQUESTED** [review]
Didn't see that this branch was meant for the other req-access branch, but if you rebase off development, we can consider bringing it in!
Also please create PR's off of the main CodeForPDX repo instead of a forked repo so it's easier to pull down and test, thanks!

**Comment by KentShikama** [comment]
@Chris-Boe Github doesn't allow me to edit the PR branch (only the base it will be merged into). In any case, I've pulled out the refactor to a different PR (see https://github.com/codeforpdx/dwellingly-app/pull/177). I'm thinking I'll just close this one and open a new one with just commit https://github.com/codeforpdx/dwellingly-app/pull/173/commits/94c75e5830307968c3f53e561b5ddb22e19a1117.

**Comment by KentShikama** [comment]
https://github.com/codeforpdx/dwellingly-app/pull/177 will also resolve the problem of having to pull down branches as the Review Apps feature on Heroku will just automatically deploy the PR branch for testing.

**Comment by Chris-Boe** [comment]
> @Chris-Boe Github doesn't allow me to edit the PR branch (only the base it will be merged into). In any case, I've pulled out the refactor to a different PR (see #177). I'm thinking I'll just close this one and open a new one with just commit [94c75e5](https://github.com/codeforpdx/dwellingly-app/commit/94c75e5830307968c3f53e561b5ddb22e19a1117).

@KentShikama Sounds good! Feel free to close this one

**Pull request closed without merging**